config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   root to: "stickers#index"
+  get '/categories/:title', to: "stickers#all"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.